tran_kien_giang/laravel_eloquent/Controller/HomeController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\CategoryController;
+use App\Models\Category;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Http\Request;
+class HomeController extends Controller
+{
+    public function home()
+    {
+        $products = DB::table('shop_products')
+        ->orderBy('sale_off', 'desc')
+        ->orderBy('product_name', 'desc')
+        ->paginate (16);
```

Comment:
```suggestion
          ->orderBy('sale_off', 'desc')
          ->orderBy('product_name', 'desc')
          ->paginate (16);
```
16 đưa vào config nhé